### ISSUE TITLE
#User Story 8: Model Performance and Evaluation

### ISSUE THREAD

**Issue by JohnStuartPhil** [JohnStuartPhil/cherryleaves#20]
As a developer, I want to provide a technical page (ML performance Metrics) to demonstrate that the model has performed to the best of its ability so that I have a record of this on the app

**Acceptance Criteria**
* AC1 - Clear graphs, charts and tables relating to the models training and performance need to be displayed

**Tasks**
* Task 1 - Show in bar chart format, the number of images in the train sets, the validation sets and the test sets
* Task 2 - Plot the model learning curves (loss and accuracy)
* Task 3 - Show a small table with loss and accuracy
